packages/ckeditor5-ckbox/src/ckboximageedit/ckboximageeditcommand.ts

```diff
@@ -234,6 +234,13 @@ export default class CKBoxImageEditCommand extends Command {
 
 		this.editor.editing.view.focus();
 
+		/**
+		 * See https://github.com/ckeditor/ckeditor5/issues/16153#top
+		 */
+		setTimeout( () => {
+			this.editor.ui.update();
+		} );
```

Comment:
Without `setTimeout` it doesn't work, so we just make sure that will be in callback stack and will be executed at the end of the execution context. I don't think cleaning up this timeout is actually necessary, because delay be default is a 0 milliseconds, it will be executed right after synchronous code and micro tasks. Also I have checked, in our codebase we have a lot of cases, when setTimeout is not cleared if delay is very low.